src/app/log-tool/system-setup/clean-data/clean-data.component.ts

```diff
@@ -94,4 +106,9 @@ export class CleanDataComponent implements OnInit {
     // });
     this.cd.detectChanges();
   }
+
+  setIntervalForSeconds(){
+    this.logToolDataService.intervalForSeconds.next(this.intervalForSeconds);
+    this.cd.detectChanges();
+  }
```

Comment:
Do we need this method? With ng template driven forms --> [(ngModel)]="intervalForSeconds" should bind to the component property when the user selects. Is there something else going on here requiring the method and detectChanges()?